### PR TITLE
Minor addition Nutraceuticals

### PR DESCRIPTION
**Pull request by RLordan**
Just some citations added and one extra line to n-3 fatty acids

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Review by rando2 — APPROVED** [review]

**Comment by agitter** [comment]
This is off topic, but I'm noting it here because this manuscript is close to submission.  This PMID/PMC reference might be wrong:
image

I believe the intended citation is `pmc:PMC2621042`, which is https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2621042/

File: content/21.nutraceuticals.md [hunk 1]
```diff
@@ -81,14 +81,15 @@ These include n-3 PUFA, zinc, vitamins C and D, and probiotics.
...
+Other, more sustainable sources of EPA and DHA include algae [@doi:10.3389/fphys.2020.00752; @doi:10.3390/fermentation4020046], which can also be exploited for their rich abundance of other bioactive compounds such as angiotensin converting enzyme (ACE) inhibitor peptides and antiviral agents including phycobiliproteins, sulfated polysaccharides, and calcium-spirulan [@doi:10.1016/j.nut.2020.111089].
 n-3 PUFAs have been investigated for many years for their therapeutic potential [@doi:10.1007/bf03257413].
 Supplementation with fish oils is generally well tolerated [@doi:10.1007/bf03257413], and intake of n-3 PUFAs through dietary sources or supplementation is specifically encouraged for vulnerable groups such as pregnant and lactating women [@url:https://www.acog.org/clinical/clinical-guidance/practice-advisory/articles/2017/01/update-on-seafood-consumption-during-pregnancy; @pmid:PMC2621042].
```
Comment (rando2) [inline content/21.nutraceuticals.md:86]:
```suggestion
Supplementation with fish oils is generally well tolerated [@doi:10.1007/bf03257413], and intake of n-3 PUFAs through dietary sources or supplementation is specifically encouraged for vulnerable groups such as pregnant and lactating women [@url:https://www.acog.org/clinical/clinical-guidance/practice-advisory/articles/2017/01/update-on-seafood-consumption-during-pregnancy; @pmc:PMC2621042].
```

**Comment by rando2** [comment]
Thank you for catching that @agitter! That was my mistake!

File: content/21.nutraceuticals.md [hunk 2]
```diff
@@ -81,14 +81,15 @@ These include n-3 PUFA, zinc, vitamins C and D, and probiotics.
...
-n-3 PUFAs have been found to mediate inflammation by influencing processes such as leukocyte chemotaxis, adhesion molecule expression, and the production of eicosanoids [@doi:10.1111/j.1365-2125.2012.04374.x].
+n-3 PUFAs have been found to mediate inflammation by influencing processes such as leukocyte chemotaxis, adhesion molecule expression, and the production of eicosanoids [@doi:10.1111/j.1365-2125.2012.04374.x; @doi:10.1016/j.bbalip.2014.08.010].
 This and other evidence indicates that n-3 PUFAs may have the capacity to modulate the adaptive immune response [@doi:10.1111/j.1365-2125.2012.04374.x; @doi:10.1039/c9fo01742a; @doi:10.1038/nrcardio.2016.103]; for example, they have been found to influence antigen presentation and the production of CD4(+) Th1 cells, among other relevant effects [@doi:10.1016/j.ejphar.2015.03.100].
+Certainly, prelimenary evidence from banked blood samples from 100 COVID-19 patients suggests that patients with a higher omega-3 index, a measure of n-3 and n-6 fatty acids in red blood cells, had a lower risk of death due to COVID-19 [@doi:10.1016/j.plefa.2021.102250].
```
Comment (rando2) [inline content/21.nutraceuticals.md:92]:
```suggestion
Certainly, preliminary evidence from banked blood samples from 100 COVID-19 patients suggests that patients with a higher omega-3 index, a measure of n-3 and n-6 fatty acids in red blood cells, had a lower risk of death due to COVID-19 [@doi:10.1016/j.plefa.2021.102250].
```

**Pull request merged**